apps/web/src/services/dashboards/shared/dashboard-variables/DashboardVariablesDropdown.vue

```diff
@@ -146,13 +146,11 @@ const handleUpdateSearchText = debounce((text: string) => {
 const filtersHelper = new QueryHelper();
 
 const getFilters = (variableProperty?: DashboardVariableSchemaProperty): QueryHelper['apiQuery']['filter']|undefined => {
-    filtersHelper.setFilters([]);
```

Comment:
issue: I think this must be exist because `getFilters` is called every time at `loadSearchResourceOptions`.
That means, the previous filters can be used at `loadSearchResourceOptions`.

Reply:
I fixed it, thanks!